packages/app/src/systems/Account/components/AccountList/AccountList.tsx

```diff
@@ -32,6 +32,13 @@ export function AccountList({
     setShowHidden((s) => !s);
   }
 
+  const collator = new Intl.Collator(undefined, {
+    numeric: true,
+    sensitivity: "base"
+  });
+
+  accounts = accounts.sort((a, b) => collator.compare(a.name, b.name));
```

Comment:
can we move this to `AccountService.getAccounts`? 
so we fix the issue at the best place where the incorrect order is being returned

https://github.com/FuelLabs/fuels-wallet/blob/4fe865a6bcf4d5969b15e5e95b93b16882fa0439/packages/app/src/systems/Account/services/account.ts#L66-L70

you'll need to use `toArray()` instead of `toCollection()` in order to run `.sort()` on it (and that `sortBy('name')` can be removed).

You’ll need to use `toArray()` instead of `toCollection()` in order to run `.sort()` on it, and `sortBy('name')` will be unnecessary with this approach.


I think this approach might work:
```tsx
static async getAccounts() {
  return db.transaction('r', db.accounts, async () => {
    const accounts = await db.accounts.toArray();
    return accounts.sort((a, b) => a.name.localeCompare(b.name, undefined, { numeric: true, sensitivity: 'base' }));
  });
}
```